Examples/BookingSystem.AspNetFramework/Stores/IdempotencyStore.cs

```diff
@@ -0,0 +1,25 @@
+using OpenActive.Server.NET.OpenBookingHelper;
```

Comment:
This is a hangover from a previous PR (https://github.com/openactive/OpenActive.Server.NET/pull/207) where the Core code wasn't copied over to Framework